components/admin_console/admin_definition.jsx

```diff
@@ -1095,7 +1095,33 @@ export default {
                             placeholder: 'admin.service.corsEx',
                             placeholder_default: 'http://example.com',
                             help_text: 'admin.service.corsDescription',
-                            help_text_default: 'Enable HTTP Cross origin request from a specific domain. Use "*" if you want to allow CORS from any domain or leave it blank to disable it.',
+                            help_text_default: 'Enable HTTP Cross origin request from a specific domain. Use "*" if you want to allow CORS from any domain or leave it blank to disable it. Should not be set to "*" in production.',
+                        },
+                        {
+                            type: Constants.SettingsTypes.TYPE_TEXT,
+                            key: 'ServiceSettings.CorsExposedHeaders',
+                            label: 'admin.service.corsExposedHeadersTitle',
+                            label_default: 'CORS Exposed Headers:',
+                            placeholder: 'admin.service.corsHeadersEx',
+                            placeholder_default: 'X-My-Header',
+                            help_text: 'admin.service.corsExposedHeadersDescription',
+                            help_text_default: 'Whitelist of headers that will be accessible to the requester.',
+                        },
+                        {
+                            type: Constants.SettingsTypes.TYPE_BOOL,
+                            key: 'ServiceSettings.CorsAllowCredentials',
+                            label: 'admin.service.corsAllowCredentialsLabel',
+                            label_default: 'CORS Allow Credentials:',
+                            help_text: 'admin.service.corsAllowCredentialsDescription',
+                            help_text_default: 'When true requests that pass validation will include the Access-Control-Allow-Credentials header.',
```

Comment:
Perhaps, `When true, requests that...` to avoid parsing as `true requests`?

Reply:
Fixed.